lib/rubocop/cop/lint/hash_each.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for uses of `each_key` & `each_value` Hash methods.
+      class HashEachMethods < Cop
+        include UnusedArgument
+
+        MSG = 'Use `%s` instead of `%s.%s`.'
+
+        EACH_METHODS = [:keys, :values]
+
+        def on_block(node)
+          _block_method, args, = *node
+          @nargs = args.child_nodes.size
```

Comment:
Using an instance variable like `@nargs` to pass a value down into `on_send` is problematic, since the value will stick around and could be reused on a completely different `send` node. It would be better to just recurse down into the `send` node (which is the first child of the `block` node) from here.